.github/workflows/main.yml

```diff
@@ -40,9 +40,12 @@ jobs:
         if: matrix.os == 'macos-latest'
       - name: Install wasmtime for tests
         run: curl -f -L --retry 5 https://wasmtime.dev/install.sh | bash -s -- --version v2.0.2
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
         with:
-          submodules: true
+          fetch-depth: 0
+      # Bump depth (or even better, submodule), in case of "error:
+      # Server does not allow request for unadvertised object"
```

Comment:
How about:

```
# We can't use `--depth 1` here sadly because the GNU config submodule is not pinned to a particular tag/branch. 
```

I do worry that this hardcoded value of 16 could become out-of-date as the config.git repository receives more commits, but maybe we just deal with that when it happens.

Reply:
Done, thanks for the addition. I've also bumped `config.git`, and this repo updates rather slowly, so we're pretty unlikely to bump into this issue for a very long time.